plugins/defaultplugins/ifplugin/interface_config.go

```diff
@@ -99,6 +104,8 @@ func (plugin *InterfaceConfigurator) Init(swIfIndexes ifaceidx.SwIfIndexRW, mtu
 	plugin.afPacketConfigurator = &AFPacketConfigurator{Logger: plugin.Log, Linux: plugin.Linux, SwIfIndexes: plugin.swIfIndexes, Stopwatch: plugin.Stopwatch}
 	plugin.afPacketConfigurator.Init(plugin.vppCh)
 
+	plugin.unnumberedIfaces = make(map[string]string)
```

Comment:
I would use name
unnumberedIfacesToIfaces it means maping from what TO what

Reply:
renamed uIfaceCache. More information is written in the comment where the var is defined